openshift/ci/tests/e2e-ansible-scaffolding.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+
+source hack/lib/common.sh
+
+set -eux
+
+component="osdk-ansible-e2e"
+eval IMAGE=$IMAGE_FORMAT
+ROOTDIR="$(pwd)"
+GOTMP="$(mktemp -d -p $GOPATH/src)"
+trap_add 'rm -rf $GOTMP' EXIT
+
+mkdir -p $ROOTDIR/bin
+export PATH=$ROOTDIR/bin:$PATH
+
+# Install kubectl client
+if ! [ -x "$(command -v kubectl)" ]; then
+    curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.21.2/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl bin/
+fi
+
+# Install oc client
+if ! [ -x "$(command -v oc)" ]; then
+    OPENSHIFT_CLIENT_VERSION="4.16.0"
+    curl -Lo oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OPENSHIFT_CLIENT_VERSION/openshift-client-linux-$OPENSHIFT_CLIENT_VERSION.tar.gz
```

Comment:
Better to use curly braces for the shell parameter expansion to avoid issues and confusion: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

```suggestion
    curl -Lo oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OPENSHIFT_CLIENT_VERSION}/openshift-client-linux-${OPENSHIFT_CLIENT_VERSION}.tar.gz
```

Reply:
Thanks for the suggestion, updated to use `${parameter}`